ampeg/src/test/java/ManualTestCases.java

```diff
@@ -0,0 +1,73 @@
+import static org.testng.Assert.assertEquals;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.text.PDFTextStripper;
+
+public class ManualTestCases extends AmpegTests{
+	@Test
+	public void verifyFaqPageTitle() {
+		var expectedPageTitle = "Frequently Asked Questions (FAQs)";
+		
+		var pageTitle = new HomePage(this.driver)
+				.clickProductsLink()
+				.clickContactLink()
+				.clickFaqLink()
+				.getPageTitle();
+		
+		assertEquals(expectedPageTitle, pageTitle, "\"Frequently Asked Questions (FAQs)\" should be displayed as the title.");
+	}
+	
+	@Test
+	public void navigateToSvt810eThroughJustinPearson() {
+		var expectedUrl = "https://ampeg.com/products/classic/svt810e/";
+		
+		var url = new HomePage(this.driver)
+				.clickArtistsLink()
+				.clickJustinPearsonLink()
+				.clickSvt810eLink()
+				.driver.getCurrentUrl();
```

Comment:
.getCurrentUrl( ) could be moved to the Page Object base since it should work on all Pages.

Any place you see WebDriver usage at the TestClass layer is a code smell and should be investigated.